addons/web/static/src/legacy/xml/colorpicker.xml

```diff
@@ -25,7 +25,7 @@
             <div class="o_rgba_div px-1 d-flex align-items-baseline">
                 <input type="text" t-attf-class="o_red_input {{input_classes}}" data-color-method="rgb" size="1"/>
                 <input type="text" t-attf-class="o_green_input {{input_classes}}" data-color-method="rgb" size="1"/>
-                <input type="text" t-attf-class="o_blue_input me-0 {{input_classes}}" data-color-method="rgb" size="1"/>
+                <input type="text" t-attf-class="o_blue_input {{input_classes}}" data-color-method="rgb" size="1"/>
```

Comment:
- Commit message:
  - "vaue" :grimacing: A nice idea would be to use a spellchecker to avoid that kind of mistake :wink: 
  - At the end of the commit message, add `task-XXX` where `XXX` is the Odoo task id
- Indeed, that `me-0` is weird but:
  - In stable, we cannot *remove/change* things from XML templates. You can *add* if necessary but not *remove/change*. Why? Some custo might be targeting this. Maybe someone actually made a custo to fix exactly this, who knows. "xpath the input in o_rgba_div which has the class me-0 and remove the class"... if you suddenly remove the class yourself, you will make that custo crash.
  - A workaround would be to make the fix in CSS or JS... but in this case: it's not worth it. It's not a bug worth solving in stable versions: put the PR back to draft (important to not ping codeowners), rebase your work on master*, push-force, retarget your PR to master*, then mark it ready to review again.

\* you could target 18.0. It's all a matter of risk/value ratio. In 16.0, the risk is quite small but the value is really small too => not worth it. In master, the risk is 0 => worth it. In 18.0, people likely don't have that kind of custo yet, the risk is close to 0 => might be worth it. I'd say let's target 18.0.